examples/sunlinsol/ginkgo/CMakeLists.txt

```diff
@@ -16,15 +16,15 @@
 # 'develop' for examples excluded from 'make test' in releases
 set(examples
   "test_sunlinsol_ginkgo.cpp\;bicg csr 100 0 100 0\;"
-  "test_sunlinsol_ginkgo.cpp\;bicg dense 100 0 100 0\;"
+#  "test_sunlinsol_ginkgo.cpp\;bicg dense 100 0 100 0\;"
   "test_sunlinsol_ginkgo.cpp\;bicgstab csr 100 0 100 0\;"
-  "test_sunlinsol_ginkgo.cpp\;bicgstab dense 100 0 100 0\;"
+#  "test_sunlinsol_ginkgo.cpp\;bicgstab dense 100 0 100 0\;"
   "test_sunlinsol_ginkgo.cpp\;cgs csr 100 0 100 0\;"
-  "test_sunlinsol_ginkgo.cpp\;cgs dense 100 0 100 0\;"
+#  "test_sunlinsol_ginkgo.cpp\;cgs dense 100 0 100 0\;"
   "test_sunlinsol_ginkgo.cpp\;gmres csr 100 0 100 0\;"
-  "test_sunlinsol_ginkgo.cpp\;gmres dense 100 0 100 0\;"
+#  "test_sunlinsol_ginkgo.cpp\;gmres dense 100 0 100 0\;"
   "test_sunlinsol_ginkgo.cpp\;idr csr 100 0 100 0\;"
-  "test_sunlinsol_ginkgo.cpp\;idr dense 100 0 100 0\;"
+#  "test_sunlinsol_ginkgo.cpp\;idr dense 100 0 100 0\;"
```

Comment:
Why are the dense tests commented out?

Reply:
https://github.com/LLNL/sundials/pull/276#issuecomment-1572551400

They fail with `subprocess aborted`

Reply:
IIRC, the released version of Ginkgo doesn't support some operation these needed.

Reply:
Now that you say it I do recall that being the issue